CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ### Enhancements
 
 * Updated documentation: Added back support doc types for partitioning, more Python codes in the API page,  RAG definition, and use case.
+* Updated `is_possible_title` detection rule
```

Comment:
maybe worth writing a bit more on what changes are introduced?

Reply:
On it!